RecoMET/METPUSubtraction/src/PFMETAlgorithmMVA.cc

```diff
@@ -224,8 +224,8 @@ void PFMETAlgorithmMVA::evaluateMVA()
   mvaOutputU_     = GetResponse(mvaReaderU_, varForU_);
   var_["RecoilCor_U"] = var_["particleFlow_U"] * mvaOutputU_;
   var_["RecoilCor_UPhi"] = var_["PhiCor_UPhi"];
-  mvaOutputCovU1_ = GetResponse(mvaReaderCovU1_, varForCovU1_)* mvaOutputU_ * var_["particleFlow_U"];
-  mvaOutputCovU2_ = GetResponse(mvaReaderCovU2_, varForCovU2_)* mvaOutputU_ * var_["particleFlow_U"];
+  mvaOutputCovU1_ = std::pow(GetResponse(mvaReaderCovU1_, varForCovU1_)* mvaOutputU_ * var_["particleFlow_U"], 2);
+  mvaOutputCovU2_ = std::pow(GetResponse(mvaReaderCovU2_, varForCovU2_)* mvaOutputU_ * var_["particleFlow_U"], 2);
```

Comment:
Hi,

yes it was. I already informed you about that on 19.11.2015 by mail and 
did some commits to the cms-met repository. I'll delete the 
corresponding branches now since they are obviously becoming obsolete.

Cheers,
Raphael

On 19.01.2016 16:57, mmarionncern wrote:
>
> In RecoMET/METPUSubtraction/src/PFMETAlgorithmMVA.cc 
> <https://github.com/cms-met/cmssw/pull/9#discussion_r50130218>:
>
> > @@ -224,8 +224,8 @@ void PFMETAlgorithmMVA::evaluateMVA()
> >    mvaOutputU_     = GetResponse(mvaReaderU_, varForU_);
> >    var_["RecoilCor_U"] = var_["particleFlow_U"] * mvaOutputU_;
> >    var_["RecoilCor_UPhi"] = var_["PhiCor_UPhi"];
> > -  mvaOutputCovU1_ = GetResponse(mvaReaderCovU1_, varForCovU1_)* mvaOutputU_ * var_["particleFlow_U"];
> > -  mvaOutputCovU2_ = GetResponse(mvaReaderCovU2_, varForCovU2_)* mvaOutputU_ * var_["particleFlow_U"];
> > +  mvaOutputCovU1_ = std::pow(GetResponse(mvaReaderCovU1_, varForCovU1_)* mvaOutputU_ * var_["particleFlow_U"], 2);
> > +  mvaOutputCovU2_ = std::pow(GetResponse(mvaReaderCovU2_, varForCovU2_)* mvaOutputU_ * var_["particleFlow_U"], 2);
>
> was it a bug to not have it before?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-met/cmssw/pull/9/files#r50130218>.
>


-- 
Raphael Friese

Karlsruhe Institute of Technology (KIT)
Kaiserstraße 12
Building 30.23 / R 8-22
76131 Karlsruhe, Germany
Phone: +49 721 608-47243

CERN
Building 32-4 B06
Phone: +41 22 76-78267

Email: Raphael.Friese@cern.ch